target/linux/ramips/dts/WD03.dts

```diff
@@ -0,0 +1,130 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "WD03", "ralink,mt7620a-soc";
```

Comment:
it has to be `compatible = "<manufacture>,<boardname>", "ralink,mt7620a-soc";` check the other dts files for examples